lib/ssh_scan/banner.rb

```diff
@@ -29,6 +29,8 @@ def ssh_lib_guess()
         return SSHScan::SSHLib::CiscoSSH.new()
       when /ROS/i
         return SSHScan::SSHLib::ROSSSH.new()
+      when /DOPRASSH/i
```

Comment:
@agaurav77 yeah, scans.io + Shodan should give us a pretty good spread of various unique fingerprints we could/would support in ssh_scan.